### DISCUSSION TITLE
Style of Parameters and Return section of numpy docstring parsed with numpydoc

### DISCUSSION THREAD

**Discussion by mscheltienne** [pradyunsg/furo#505] [Potential Issue] [answered]
Hello!

First, thanks for this fantastic, simple, and super clean theme! I've been playing with it and with my `sphinx` configuration for a while on this sandbox repository: https://github.com/mscheltienne/simple-stimuli

A numpy docstring is composed of several sections: `Parameters`, `Returns`, `Notes`, `Examples`, `References`, .. for instance, this [method](https://github.com/mscheltienne/simple-stimuli/blob/819399b21c14bc2c9f7b7cb29070325b276fb889/stimuli/visuals/base.py#L72-L94).

```
    @fill_doc
    def draw_background(self, color: Union[str, Tuple[int, int, int]]) -> None:
        """Draw a uniform single color background.

        Replace all the pixels with this color, thus this method erases any
        prior work.

        Parameters
        ----------
        color : str | tuple
            Color used to draw the background.

        Notes
        -----
        %(visual_color)s
        """
        color = BaseVisual._check_color(color)
        self._img = np.full(
            (self._window_height, self._window_width, 3),
            fill_value=color,
            dtype=np.uint8,
        )
        self._background = color
```

The section titles are not rendered with the same font/size. For the example above, you can find the render [here](https://mscheltienne.github.io/simple-stimuli/dev/api/generated/stimuli.visuals.Cross.html#stimuli.visuals.Cross.draw_background).

![image](https://user-images.githubusercontent.com/73893616/184636137-f6b4a82f-b43b-4a70-ade0-530c96b75e0b.png)

I would like to have all section titles rendered in the same way. Currently, both `Parameters` and `Returns` are rendered differently, and IMO, they are rendered too small compared to the elements listed in each section.

**Comment by pradyunsg** [comment]
Please share a link to the rendered documentation, as available somewhere online (or instructions to build the docs).

**Reply by mscheltienne** [reply]
There is a link, on the word 'here': https://mscheltienne.github.io/simple-stimuli/dev/api/generated/stimuli.visuals.Cross.html#stimuli.visuals.Cross.draw_background

To build the documentation, clone the repository, install the dependencies with `pip install .[doc]` and build the documentation with `make html` from the `doc` directory. The usual ;) The CI building the documentation is defined here: https://github.com/mscheltienne/simple-stimuli/blob/main/.github/workflows/doc.yml

**Comment by pradyunsg** [comment, marked as answer]
This one specific topic seems to have slipped through the cracks, but it looks like this has been resolved by handling `.rubric` classes more consistently with the field headers. :)
